lib/python/satos_payload_sdk/antaris_api_client.py

```diff
@@ -151,10 +188,20 @@ def api_pa_pc_create_channel_common(secure, callback_func_list):
             quit()
 
         credentials = grpc.ssl_channel_credentials(root_certs)
+        """
+        if api_common.g_KEEPALIVE_ENABLE == '0':
+            channel = grpc.secure_channel( pc_endpoint , credentials)
+        else:
+            channel = grpc.secure_channel( pc_endpoint , credentials, options=server_options)
+        """
```

Comment:
Is this commented-out code necessary?

Reply:
Keepalive is only send by client i.e. remote application. Hence channel code on client side is commented now. But, if we decide to enable server side keepalive, then this code will be helpful